nncf/common/tensor_statistics/collectors.py

```diff
@@ -349,6 +349,41 @@ def _max_aggregate(self):
         return self._tensor_processor.mean(stacked_max, axis=0)
 
 
+class MeanStatisticCollector(OfflineTensorStatisticCollector):
+    """
+    Collector aggregates mean per axis.
+    """
+
+    def __init__(self,
+                 reduction_shape: ReductionShape,
+                 num_samples: int = None,
+                 window_size: int = None):
```

Comment:
```suggestion
                 num_samples: Optional[int] = None,
                 window_size: Optional[int] = None):
```

Is it really optional arguments?